qiskit_nature/second_q/circuit/library/ansatzes/succd.py

```diff
@@ -106,27 +108,46 @@ def include_singles(self, include_singles: Tuple[bool, bool]) -> None:
         """Sets whether to include single excitations."""
         self._include_singles = include_singles
 
+    @property
+    def mirror(self) -> bool:
+        """Whether include the symmetrically mirrored double excitations (SUCC_full)."""
+        return self._mirror
+
+    @mirror.setter
+    def mirror(self, mirror: bool) -> None:
+        """Sets whether include the symmetrically mirrored double excitations (SUCC_full)."""
+        self._mirror = mirror
+
+    def _filter_operators(self, operators):
+        excitation_dict = self._excitations_dict
+        valid_operators, valid_excitations = [], []
+        for op, ex in zip(operators, excitation_dict.values()):
```

Comment:
This implementation now looks good. But maybe you can just do this:
```suggestion
        valid_operators, valid_excitations = [], []
        for op, ex in zip(operators, self._excitation_dict.values()):
```